### ISSUE TITLE
Change range but not drag the entire item

### ISSUE THREAD

**Issue by artaxa** [almende/vis#422]
Is it possible to only allow dragging the start or the end of a (range)item to change it's start- or end-date, but not allowing that the whole item is dragged to another moment in time?

**Comment by AlexDM0** [comment]
Could we rephrase this to a feature request or can we close the issue?

**Comment by josdejong** [comment]
I think this is just an issue, not a feature request

**Comment by AlexDM0** [comment]
Is it still open?

**Comment by josdejong** [comment]
yes I think so

**Comment by artaxa** [comment]
Sorry... It's not a feature request (yet)
